SQL/New_patches/2019-11-27-RemoveOldConfigSettings.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM ConfigSettings WHERE Name='Logs';
+DELETE FROM ConfigSettings WHERE Name='css';
+DELETE FROM ConfigSettings WHERE Name='issue_tracker_url';
+DELETE FROM ConfigSettings WHERE Name='FileGroup';
```

Comment:
This (FileGroup) appears to not be unused, you're just deleting the code that uses it and removing the feature in this PR.

Reply:
Where is it used?

Reply:
In the 5 lines deleted in  php/libraries/File_Upload.class.inc  in this PR.

Reply:
OK good point. I haven't really looked at this since November. I'll revert this.